package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "last-train-to-nodeville",
+  "version": "1.0.0",
+  "description": "## Instructions",
+  "main": "index.js",
```

Comment:
there are dev dependencies and would be better specified there.  this can be done via `npm install --save-dev`

Reply:
executed npm install --save-dev